src/components/MessageForm/MessageForm.tsx

```diff
@@ -52,7 +52,7 @@ const MessageForm: React.FC<RouteComponentProps> = (props) => {
           <textarea id="employer_email" className="applicant-input" onChange={handleMessageChange}/>
           <label htmlFor="body">Your message here:</label>
           <textarea id="body" className="applicant-input" rows={3} onChange={handleMessageChange}/>
-          <input type="submit" value="Send" className="cta-button"/>
+          <button className="cta-button" value="Send" placeholder="send">Send</button>
```

Comment:
I think this may be why the test is not working. We will need this to remain an input with the type submit so that when the button is clicked, the `onSubmit` in `<form>` is triggered. We can then test that the fetch method `sendMessage` was called 1 time. We would need to mock the response of that API call as well at the beginning of the test.

I understand that changing this back to the input will break the other two tests but they can be fixed by just replacing the line where they are declared to `const sendButton = screen.getByRole("button", { name: /submit/i })`.

Let me know what you think!

Reply:
That did not work for some strange reason? I was going to look w fresher eyes today!  The other tests pass because the placeholder is present. I meant to add an aria label here, so will fix that. 

I am not sure why the test does not like a fire event just yet either. I wanted to push this up so all our tests are on the table so to speak!

Reply:
We can leave it as a button going forward if you'd like. Here is what I changed to make it pass:

```
// maybe we could make this "Should fire the correct method when send button clicked"
  it('Send should fire an event', async () => {
		mocked(sendMessage).mockImplementation(() =>
			Promise.resolve({})
		)
		const routeComponentPropsMock = {
			history: {} as any,
			location: {} as any,
			match: {
				isExact: true,
				params: {id: 1},
				path: "",
				url: ""
			}
		}

		const showMessageForm = jest.fn()
		const messageForm = true

		render(<MemoryRouter><MessageFormContext.Provider value={{ messageForm, showMessageForm }}><MessageForm {...routeComponentPropsMock} /></MessageFormContext.Provider></MemoryRouter>)
		
		const employerName = screen.getByLabelText(/name/i)
		const employerEmail = screen.getByLabelText(/email/i)
		const message = screen.getByLabelText(/message/i)
		
		fireEvent.change(employerName, { target: { value: /google/i } })
		fireEvent.change(employerEmail, { target: { value: /google@gmail.com/i } })
		fireEvent.change(message, { target: { value: /please work for us!/i } })

    const sendButton = screen.getByRole("button", { name: /send/i })
		fireEvent.click(sendButton)
		
		expect(sendMessage).toBeCalledTimes(1)
	})
```

Reply:
Erin, thanks for the suggestions. I attempted to use act() and account for async possibilities, but we still have a warning for this. There is an issue to revisit.